src/data/set/basic.lean

```diff
@@ -1737,6 +1737,12 @@ by { ext, simp }
 @[simp] theorem preimage_inr_range_inl : sum.inr ⁻¹' range (sum.inl : α → α ⊕ β) = ∅ :=
 by { ext, simp }
 
+@[simp] lemma compl_range_inl : (range (sum.inl : α → α ⊕ β))ᶜ = range (sum.inr : β → α ⊕ β) :=
```

Comment:
We have an `is_compl` lemma above. I would use it (even if the proof is a bit longer).